python/llm/src/bigdl/llm/optimize.py

```diff
@@ -138,6 +138,16 @@ def load_low_bit(model, model_path):
                       "model should be a instance of "
                       f"`torch.nn.Module`, but got {type(model)} at last.")
     if low_bit:
+        # a creator
+        if is_creator:
```

Comment:
`is_creator` seems like a flag but where get it? and the following `model_or_creator` also is confusing...

Reply:
Yes this is a conflict, and has been fixed.